rcute_cozmars/button.py

```diff
@@ -1,57 +1,56 @@
-from . import util
+from. import util
 
 class Button(util.Component):
-    """按钮
+    """ Button
```

Comment:
Do you think it is better without doc string in cases where function names or class names are straight forward？it looks silly to put the same word under def.